PID 1249
INFO  Start processing
INFO  Hexo is running at http://localhost:4000 . Press Ctrl+C to stop.
INFO  Start processing
INFO  Hexo is running at http://localhost:4000 . Press Ctrl+C to stop.
(node:15116) [DEP0066] DeprecationWarning: OutgoingMessage.prototype._headers is deprecated
EEE Error: spawn UNKNOWN
    at ChildProcess.spawn (internal/child_process.js:394:11)
    at spawn (child_process.js:540:9)
    at module.exports (C:\data\Treeeeeeee.github.io\blog\node_modules\hexo-admin\deploy.js:14:14)
    at C:\data\Treeeeeeee.github.io\blog\node_modules\hexo-admin\api.js:413:7
    at C:\data\Treeeeeeee.github.io\blog\node_modules\hexo-admin\api.js:141:7
    at call (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:239:7)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:183:5)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at C:\data\Treeeeeeee.github.io\blog\node_modules\body-parser\lib\read.js:130:5
    at invokeCallback (C:\data\Treeeeeeee.github.io\blog\node_modules\raw-body\index.js:224:16)
    at done (C:\data\Treeeeeeee.github.io\blog\node_modules\raw-body\index.js:213:7)
    at IncomingMessage.onEnd (C:\data\Treeeeeeee.github.io\blog\node_modules\raw-body\index.js:273:7)
    at IncomingMessage.emit (events.js:215:7)
    at endReadableNT (_stream_readable.js:1183:12)
    at processTicksAndRejections (internal/process/task_queues.js:80:21) {
  errno: 'UNKNOWN',
  code: 'UNKNOWN',
  syscall: 'spawn'
}
EEE Error: spawn UNKNOWN
    at ChildProcess.spawn (internal/child_process.js:394:11)
    at spawn (child_process.js:540:9)
    at module.exports (C:\data\Treeeeeeee.github.io\blog\node_modules\hexo-admin\deploy.js:14:14)
    at C:\data\Treeeeeeee.github.io\blog\node_modules\hexo-admin\api.js:413:7
    at C:\data\Treeeeeeee.github.io\blog\node_modules\hexo-admin\api.js:141:7
    at call (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:239:7)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:183:5)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at C:\data\Treeeeeeee.github.io\blog\node_modules\body-parser\lib\read.js:130:5
    at invokeCallback (C:\data\Treeeeeeee.github.io\blog\node_modules\raw-body\index.js:224:16)
    at done (C:\data\Treeeeeeee.github.io\blog\node_modules\raw-body\index.js:213:7)
    at IncomingMessage.onEnd (C:\data\Treeeeeeee.github.io\blog\node_modules\raw-body\index.js:273:7)
    at IncomingMessage.emit (events.js:215:7)
    at endReadableNT (_stream_readable.js:1183:12)
    at processTicksAndRejections (internal/process/task_queues.js:80:21) {
  errno: 'UNKNOWN',
  code: 'UNKNOWN',
  syscall: 'spawn'
}
EEE Error: spawn UNKNOWN
    at ChildProcess.spawn (internal/child_process.js:394:11)
    at spawn (child_process.js:540:9)
    at module.exports (C:\data\Treeeeeeee.github.io\blog\node_modules\hexo-admin\deploy.js:14:14)
    at C:\data\Treeeeeeee.github.io\blog\node_modules\hexo-admin\api.js:413:7
    at C:\data\Treeeeeeee.github.io\blog\node_modules\hexo-admin\api.js:141:7
    at call (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:239:7)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:183:5)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at C:\data\Treeeeeeee.github.io\blog\node_modules\body-parser\lib\read.js:130:5
    at invokeCallback (C:\data\Treeeeeeee.github.io\blog\node_modules\raw-body\index.js:224:16)
    at done (C:\data\Treeeeeeee.github.io\blog\node_modules\raw-body\index.js:213:7)
    at IncomingMessage.onEnd (C:\data\Treeeeeeee.github.io\blog\node_modules\raw-body\index.js:273:7)
    at IncomingMessage.emit (events.js:215:7)
    at endReadableNT (_stream_readable.js:1183:12)
    at processTicksAndRejections (internal/process/task_queues.js:80:21) {
  errno: 'UNKNOWN',
  code: 'UNKNOWN',
  syscall: 'spawn'
}
EEE Error: spawn UNKNOWN
    at ChildProcess.spawn (internal/child_process.js:394:11)
    at spawn (child_process.js:540:9)
    at module.exports (C:\data\Treeeeeeee.github.io\blog\node_modules\hexo-admin\deploy.js:14:14)
    at C:\data\Treeeeeeee.github.io\blog\node_modules\hexo-admin\api.js:413:7
    at C:\data\Treeeeeeee.github.io\blog\node_modules\hexo-admin\api.js:141:7
    at call (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:239:7)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:183:5)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at C:\data\Treeeeeeee.github.io\blog\node_modules\body-parser\lib\read.js:130:5
    at invokeCallback (C:\data\Treeeeeeee.github.io\blog\node_modules\raw-body\index.js:224:16)
    at done (C:\data\Treeeeeeee.github.io\blog\node_modules\raw-body\index.js:213:7)
    at IncomingMessage.onEnd (C:\data\Treeeeeeee.github.io\blog\node_modules\raw-body\index.js:273:7)
    at IncomingMessage.emit (events.js:215:7)
    at endReadableNT (_stream_readable.js:1183:12)
    at processTicksAndRejections (internal/process/task_queues.js:80:21) {
  errno: 'UNKNOWN',
  code: 'UNKNOWN',
  syscall: 'spawn'
}
EEE Error: spawn UNKNOWN
    at ChildProcess.spawn (internal/child_process.js:394:11)
    at spawn (child_process.js:540:9)
    at module.exports (C:\data\Treeeeeeee.github.io\blog\node_modules\hexo-admin\deploy.js:14:14)
    at C:\data\Treeeeeeee.github.io\blog\node_modules\hexo-admin\api.js:413:7
    at C:\data\Treeeeeeee.github.io\blog\node_modules\hexo-admin\api.js:141:7
    at call (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:239:7)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:183:5)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at C:\data\Treeeeeeee.github.io\blog\node_modules\body-parser\lib\read.js:130:5
    at invokeCallback (C:\data\Treeeeeeee.github.io\blog\node_modules\raw-body\index.js:224:16)
    at done (C:\data\Treeeeeeee.github.io\blog\node_modules\raw-body\index.js:213:7)
    at IncomingMessage.onEnd (C:\data\Treeeeeeee.github.io\blog\node_modules\raw-body\index.js:273:7)
    at IncomingMessage.emit (events.js:215:7)
    at endReadableNT (_stream_readable.js:1183:12)
    at processTicksAndRejections (internal/process/task_queues.js:80:21) {
  errno: 'UNKNOWN',
  code: 'UNKNOWN',
  syscall: 'spawn'
}
EEE Error: spawn UNKNOWN
    at ChildProcess.spawn (internal/child_process.js:394:11)
    at spawn (child_process.js:540:9)
    at module.exports (C:\data\Treeeeeeee.github.io\blog\node_modules\hexo-admin\deploy.js:14:14)
    at C:\data\Treeeeeeee.github.io\blog\node_modules\hexo-admin\api.js:413:7
    at C:\data\Treeeeeeee.github.io\blog\node_modules\hexo-admin\api.js:141:7
    at call (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:239:7)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:183:5)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at C:\data\Treeeeeeee.github.io\blog\node_modules\body-parser\lib\read.js:130:5
    at invokeCallback (C:\data\Treeeeeeee.github.io\blog\node_modules\raw-body\index.js:224:16)
    at done (C:\data\Treeeeeeee.github.io\blog\node_modules\raw-body\index.js:213:7)
    at IncomingMessage.onEnd (C:\data\Treeeeeeee.github.io\blog\node_modules\raw-body\index.js:273:7)
    at IncomingMessage.emit (events.js:215:7)
    at endReadableNT (_stream_readable.js:1183:12)
    at processTicksAndRejections (internal/process/task_queues.js:80:21) {
  errno: 'UNKNOWN',
  code: 'UNKNOWN',
  syscall: 'spawn'
}
/c/Users/12466/AppData/Roaming/npm/hexo: line 14:  1295 Terminated              node "$basedir/node_modules/hexo-cli/bin/hexo" "$@"
INFO  Start processing
INFO  Hexo is running at http://localhost:4000 . Press Ctrl+C to stop.
(node:1524) [DEP0066] DeprecationWarning: OutgoingMessage.prototype._headers is deprecated
EEE Error: spawn UNKNOWN
    at ChildProcess.spawn (internal/child_process.js:394:11)
    at spawn (child_process.js:540:9)
    at module.exports (C:\data\Treeeeeeee.github.io\blog\node_modules\hexo-admin\deploy.js:14:14)
    at C:\data\Treeeeeeee.github.io\blog\node_modules\hexo-admin\api.js:413:7
    at C:\data\Treeeeeeee.github.io\blog\node_modules\hexo-admin\api.js:141:7
    at call (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:239:7)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:183:5)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at C:\data\Treeeeeeee.github.io\blog\node_modules\body-parser\lib\read.js:130:5
    at invokeCallback (C:\data\Treeeeeeee.github.io\blog\node_modules\raw-body\index.js:224:16)
    at done (C:\data\Treeeeeeee.github.io\blog\node_modules\raw-body\index.js:213:7)
    at IncomingMessage.onEnd (C:\data\Treeeeeeee.github.io\blog\node_modules\raw-body\index.js:273:7)
    at IncomingMessage.emit (events.js:215:7)
    at endReadableNT (_stream_readable.js:1183:12)
    at processTicksAndRejections (internal/process/task_queues.js:80:21) {
  errno: 'UNKNOWN',
  code: 'UNKNOWN',
  syscall: 'spawn'
}
EEE Error: spawn UNKNOWN
    at ChildProcess.spawn (internal/child_process.js:394:11)
    at spawn (child_process.js:540:9)
    at module.exports (C:\data\Treeeeeeee.github.io\blog\node_modules\hexo-admin\deploy.js:14:14)
    at C:\data\Treeeeeeee.github.io\blog\node_modules\hexo-admin\api.js:413:7
    at C:\data\Treeeeeeee.github.io\blog\node_modules\hexo-admin\api.js:141:7
    at call (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:239:7)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:183:5)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at C:\data\Treeeeeeee.github.io\blog\node_modules\body-parser\lib\read.js:130:5
    at invokeCallback (C:\data\Treeeeeeee.github.io\blog\node_modules\raw-body\index.js:224:16)
    at done (C:\data\Treeeeeeee.github.io\blog\node_modules\raw-body\index.js:213:7)
    at IncomingMessage.onEnd (C:\data\Treeeeeeee.github.io\blog\node_modules\raw-body\index.js:273:7)
    at IncomingMessage.emit (events.js:215:7)
    at endReadableNT (_stream_readable.js:1183:12)
    at processTicksAndRejections (internal/process/task_queues.js:80:21) {
  errno: 'UNKNOWN',
  code: 'UNKNOWN',
  syscall: 'spawn'
}
EEE Error: spawn UNKNOWN
    at ChildProcess.spawn (internal/child_process.js:394:11)
    at spawn (child_process.js:540:9)
    at module.exports (C:\data\Treeeeeeee.github.io\blog\node_modules\hexo-admin\deploy.js:14:14)
    at C:\data\Treeeeeeee.github.io\blog\node_modules\hexo-admin\api.js:413:7
    at C:\data\Treeeeeeee.github.io\blog\node_modules\hexo-admin\api.js:141:7
    at call (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:239:7)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:183:5)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at C:\data\Treeeeeeee.github.io\blog\node_modules\body-parser\lib\read.js:130:5
    at invokeCallback (C:\data\Treeeeeeee.github.io\blog\node_modules\raw-body\index.js:224:16)
    at done (C:\data\Treeeeeeee.github.io\blog\node_modules\raw-body\index.js:213:7)
    at IncomingMessage.onEnd (C:\data\Treeeeeeee.github.io\blog\node_modules\raw-body\index.js:273:7)
    at IncomingMessage.emit (events.js:215:7)
    at endReadableNT (_stream_readable.js:1183:12)
    at processTicksAndRejections (internal/process/task_queues.js:80:21) {
  errno: 'UNKNOWN',
  code: 'UNKNOWN',
  syscall: 'spawn'
}
EEE Error: spawn UNKNOWN
    at ChildProcess.spawn (internal/child_process.js:394:11)
    at spawn (child_process.js:540:9)
    at module.exports (C:\data\Treeeeeeee.github.io\blog\node_modules\hexo-admin\deploy.js:14:14)
    at C:\data\Treeeeeeee.github.io\blog\node_modules\hexo-admin\api.js:413:7
    at C:\data\Treeeeeeee.github.io\blog\node_modules\hexo-admin\api.js:141:7
    at call (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:239:7)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:183:5)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at C:\data\Treeeeeeee.github.io\blog\node_modules\body-parser\lib\read.js:130:5
    at invokeCallback (C:\data\Treeeeeeee.github.io\blog\node_modules\raw-body\index.js:224:16)
    at done (C:\data\Treeeeeeee.github.io\blog\node_modules\raw-body\index.js:213:7)
    at IncomingMessage.onEnd (C:\data\Treeeeeeee.github.io\blog\node_modules\raw-body\index.js:273:7)
    at IncomingMessage.emit (events.js:215:7)
    at endReadableNT (_stream_readable.js:1183:12)
    at processTicksAndRejections (internal/process/task_queues.js:80:21) {
  errno: 'UNKNOWN',
  code: 'UNKNOWN',
  syscall: 'spawn'
}
EEE Error: spawn UNKNOWN
    at ChildProcess.spawn (internal/child_process.js:394:11)
    at spawn (child_process.js:540:9)
    at module.exports (C:\data\Treeeeeeee.github.io\blog\node_modules\hexo-admin\deploy.js:14:14)
    at C:\data\Treeeeeeee.github.io\blog\node_modules\hexo-admin\api.js:413:7
    at C:\data\Treeeeeeee.github.io\blog\node_modules\hexo-admin\api.js:141:7
    at call (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:239:7)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:183:5)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at C:\data\Treeeeeeee.github.io\blog\node_modules\body-parser\lib\read.js:130:5
    at invokeCallback (C:\data\Treeeeeeee.github.io\blog\node_modules\raw-body\index.js:224:16)
    at done (C:\data\Treeeeeeee.github.io\blog\node_modules\raw-body\index.js:213:7)
    at IncomingMessage.onEnd (C:\data\Treeeeeeee.github.io\blog\node_modules\raw-body\index.js:273:7)
    at IncomingMessage.emit (events.js:215:7)
    at endReadableNT (_stream_readable.js:1183:12)
    at processTicksAndRejections (internal/process/task_queues.js:80:21) {
  errno: 'UNKNOWN',
  code: 'UNKNOWN',
  syscall: 'spawn'
}
EEE Error: spawn UNKNOWN
    at ChildProcess.spawn (internal/child_process.js:394:11)
    at spawn (child_process.js:540:9)
    at module.exports (C:\data\Treeeeeeee.github.io\blog\node_modules\hexo-admin\deploy.js:14:14)
    at C:\data\Treeeeeeee.github.io\blog\node_modules\hexo-admin\api.js:413:7
    at C:\data\Treeeeeeee.github.io\blog\node_modules\hexo-admin\api.js:141:7
    at call (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:239:7)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:183:5)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at C:\data\Treeeeeeee.github.io\blog\node_modules\body-parser\lib\read.js:130:5
    at invokeCallback (C:\data\Treeeeeeee.github.io\blog\node_modules\raw-body\index.js:224:16)
    at done (C:\data\Treeeeeeee.github.io\blog\node_modules\raw-body\index.js:213:7)
    at IncomingMessage.onEnd (C:\data\Treeeeeeee.github.io\blog\node_modules\raw-body\index.js:273:7)
    at IncomingMessage.emit (events.js:215:7)
    at endReadableNT (_stream_readable.js:1183:12)
    at processTicksAndRejections (internal/process/task_queues.js:80:21) {
  errno: 'UNKNOWN',
  code: 'UNKNOWN',
  syscall: 'spawn'
}
EEE Error: spawn UNKNOWN
    at ChildProcess.spawn (internal/child_process.js:394:11)
    at spawn (child_process.js:540:9)
    at module.exports (C:\data\Treeeeeeee.github.io\blog\node_modules\hexo-admin\deploy.js:14:14)
    at C:\data\Treeeeeeee.github.io\blog\node_modules\hexo-admin\api.js:413:7
    at C:\data\Treeeeeeee.github.io\blog\node_modules\hexo-admin\api.js:141:7
    at call (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:239:7)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:183:5)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at C:\data\Treeeeeeee.github.io\blog\node_modules\body-parser\lib\read.js:130:5
    at invokeCallback (C:\data\Treeeeeeee.github.io\blog\node_modules\raw-body\index.js:224:16)
    at done (C:\data\Treeeeeeee.github.io\blog\node_modules\raw-body\index.js:213:7)
    at IncomingMessage.onEnd (C:\data\Treeeeeeee.github.io\blog\node_modules\raw-body\index.js:273:7)
    at IncomingMessage.emit (events.js:215:7)
    at endReadableNT (_stream_readable.js:1183:12)
    at processTicksAndRejections (internal/process/task_queues.js:80:21) {
  errno: 'UNKNOWN',
  code: 'UNKNOWN',
  syscall: 'spawn'
}
res Error: spawn hexo-deploy.sh ENOENT
    at Process.ChildProcess._handle.onexit (internal/child_process.js:264:19)
    at onErrorNT (internal/child_process.js:456:16)
    at processTicksAndRejections (internal/process/task_queues.js:80:21) {
  errno: 'ENOENT',
  code: 'ENOENT',
  syscall: 'spawn hexo-deploy.sh',
  path: 'hexo-deploy.sh',
  spawnargs: [ '' ]
} { stdout: '', stderr: '' }
res Error: spawn hexo-deploy.sh ENOENT
    at Process.ChildProcess._handle.onexit (internal/child_process.js:264:19)
    at onErrorNT (internal/child_process.js:456:16)
    at processTicksAndRejections (internal/process/task_queues.js:80:21) {
  errno: 'ENOENT',
  code: 'ENOENT',
  syscall: 'spawn hexo-deploy.sh',
  path: 'hexo-deploy.sh',
  spawnargs: [ '' ]
} { stdout: '', stderr: '' }
res Error: spawn hexo-deploy.sh ENOENT
    at Process.ChildProcess._handle.onexit (internal/child_process.js:264:19)
    at onErrorNT (internal/child_process.js:456:16)
    at processTicksAndRejections (internal/process/task_queues.js:80:21) {
  errno: 'ENOENT',
  code: 'ENOENT',
  syscall: 'spawn hexo-deploy.sh',
  path: 'hexo-deploy.sh',
  spawnargs: [ '' ]
} { stdout: '', stderr: '' }
res Error: spawn hexo-deploy.sh ENOENT
    at Process.ChildProcess._handle.onexit (internal/child_process.js:264:19)
    at onErrorNT (internal/child_process.js:456:16)
    at processTicksAndRejections (internal/process/task_queues.js:80:21) {
  errno: 'ENOENT',
  code: 'ENOENT',
  syscall: 'spawn hexo-deploy.sh',
  path: 'hexo-deploy.sh',
  spawnargs: [ '' ]
} { stdout: '', stderr: '' }
res Error: spawn hexo-deploy.sh ENOENT
    at Process.ChildProcess._handle.onexit (internal/child_process.js:264:19)
    at onErrorNT (internal/child_process.js:456:16)
    at processTicksAndRejections (internal/process/task_queues.js:80:21) {
  errno: 'ENOENT',
  code: 'ENOENT',
  syscall: 'spawn hexo-deploy.sh',
  path: 'hexo-deploy.sh',
  spawnargs: [ '' ]
} { stdout: '', stderr: '' }
res Error: spawn hexo-deploy.sh ENOENT
    at Process.ChildProcess._handle.onexit (internal/child_process.js:264:19)
    at onErrorNT (internal/child_process.js:456:16)
    at processTicksAndRejections (internal/process/task_queues.js:80:21) {
  errno: 'ENOENT',
  code: 'ENOENT',
  syscall: 'spawn hexo-deploy.sh',
  path: 'hexo-deploy.sh',
  spawnargs: [ '' ]
} { stdout: '', stderr: '' }
EEE Error: spawn UNKNOWN
    at ChildProcess.spawn (internal/child_process.js:394:11)
    at spawn (child_process.js:540:9)
    at module.exports (C:\data\Treeeeeeee.github.io\blog\node_modules\hexo-admin\deploy.js:14:14)
    at C:\data\Treeeeeeee.github.io\blog\node_modules\hexo-admin\api.js:413:7
    at C:\data\Treeeeeeee.github.io\blog\node_modules\hexo-admin\api.js:141:7
    at call (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:239:7)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:183:5)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at C:\data\Treeeeeeee.github.io\blog\node_modules\body-parser\lib\read.js:130:5
    at invokeCallback (C:\data\Treeeeeeee.github.io\blog\node_modules\raw-body\index.js:224:16)
    at done (C:\data\Treeeeeeee.github.io\blog\node_modules\raw-body\index.js:213:7)
    at IncomingMessage.onEnd (C:\data\Treeeeeeee.github.io\blog\node_modules\raw-body\index.js:273:7)
    at IncomingMessage.emit (events.js:215:7)
    at endReadableNT (_stream_readable.js:1183:12)
    at processTicksAndRejections (internal/process/task_queues.js:80:21) {
  errno: 'UNKNOWN',
  code: 'UNKNOWN',
  syscall: 'spawn'
}
EEE Error: spawn UNKNOWN
    at ChildProcess.spawn (internal/child_process.js:394:11)
    at spawn (child_process.js:540:9)
    at module.exports (C:\data\Treeeeeeee.github.io\blog\node_modules\hexo-admin\deploy.js:14:14)
    at C:\data\Treeeeeeee.github.io\blog\node_modules\hexo-admin\api.js:413:7
    at C:\data\Treeeeeeee.github.io\blog\node_modules\hexo-admin\api.js:141:7
    at call (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:239:7)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:183:5)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at C:\data\Treeeeeeee.github.io\blog\node_modules\body-parser\lib\read.js:130:5
    at invokeCallback (C:\data\Treeeeeeee.github.io\blog\node_modules\raw-body\index.js:224:16)
    at done (C:\data\Treeeeeeee.github.io\blog\node_modules\raw-body\index.js:213:7)
    at IncomingMessage.onEnd (C:\data\Treeeeeeee.github.io\blog\node_modules\raw-body\index.js:273:7)
    at IncomingMessage.emit (events.js:215:7)
    at endReadableNT (_stream_readable.js:1183:12)
    at processTicksAndRejections (internal/process/task_queues.js:80:21) {
  errno: 'UNKNOWN',
  code: 'UNKNOWN',
  syscall: 'spawn'
}
EEE Error: spawn UNKNOWN
    at ChildProcess.spawn (internal/child_process.js:394:11)
    at spawn (child_process.js:540:9)
    at module.exports (C:\data\Treeeeeeee.github.io\blog\node_modules\hexo-admin\deploy.js:14:14)
    at C:\data\Treeeeeeee.github.io\blog\node_modules\hexo-admin\api.js:413:7
    at C:\data\Treeeeeeee.github.io\blog\node_modules\hexo-admin\api.js:141:7
    at call (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:239:7)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:183:5)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at C:\data\Treeeeeeee.github.io\blog\node_modules\body-parser\lib\read.js:130:5
    at invokeCallback (C:\data\Treeeeeeee.github.io\blog\node_modules\raw-body\index.js:224:16)
    at done (C:\data\Treeeeeeee.github.io\blog\node_modules\raw-body\index.js:213:7)
    at IncomingMessage.onEnd (C:\data\Treeeeeeee.github.io\blog\node_modules\raw-body\index.js:273:7)
    at IncomingMessage.emit (events.js:215:7)
    at endReadableNT (_stream_readable.js:1183:12)
    at processTicksAndRejections (internal/process/task_queues.js:80:21) {
  errno: 'UNKNOWN',
  code: 'UNKNOWN',
  syscall: 'spawn'
}
EEE Error: spawn UNKNOWN
    at ChildProcess.spawn (internal/child_process.js:394:11)
    at spawn (child_process.js:540:9)
    at module.exports (C:\data\Treeeeeeee.github.io\blog\node_modules\hexo-admin\deploy.js:14:14)
    at C:\data\Treeeeeeee.github.io\blog\node_modules\hexo-admin\api.js:413:7
    at C:\data\Treeeeeeee.github.io\blog\node_modules\hexo-admin\api.js:141:7
    at call (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:239:7)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:183:5)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at C:\data\Treeeeeeee.github.io\blog\node_modules\body-parser\lib\read.js:130:5
    at invokeCallback (C:\data\Treeeeeeee.github.io\blog\node_modules\raw-body\index.js:224:16)
    at done (C:\data\Treeeeeeee.github.io\blog\node_modules\raw-body\index.js:213:7)
    at IncomingMessage.onEnd (C:\data\Treeeeeeee.github.io\blog\node_modules\raw-body\index.js:273:7)
    at IncomingMessage.emit (events.js:215:7)
    at endReadableNT (_stream_readable.js:1183:12)
    at processTicksAndRejections (internal/process/task_queues.js:80:21) {
  errno: 'UNKNOWN',
  code: 'UNKNOWN',
  syscall: 'spawn'
}
EEE Error: spawn UNKNOWN
    at ChildProcess.spawn (internal/child_process.js:394:11)
    at spawn (child_process.js:540:9)
    at module.exports (C:\data\Treeeeeeee.github.io\blog\node_modules\hexo-admin\deploy.js:14:14)
    at C:\data\Treeeeeeee.github.io\blog\node_modules\hexo-admin\api.js:413:7
    at C:\data\Treeeeeeee.github.io\blog\node_modules\hexo-admin\api.js:141:7
    at call (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:239:7)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:183:5)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at next (C:\data\Treeeeeeee.github.io\blog\node_modules\connect\index.js:161:14)
    at C:\data\Treeeeeeee.github.io\blog\node_modules\body-parser\lib\read.js:130:5
    at invokeCallback (C:\data\Treeeeeeee.github.io\blog\node_modules\raw-body\index.js:224:16)
    at done (C:\data\Treeeeeeee.github.io\blog\node_modules\raw-body\index.js:213:7)
    at IncomingMessage.onEnd (C:\data\Treeeeeeee.github.io\blog\node_modules\raw-body\index.js:273:7)
    at IncomingMessage.emit (events.js:215:7)
    at endReadableNT (_stream_readable.js:1183:12)
    at processTicksAndRejections (internal/process/task_queues.js:80:21) {
  errno: 'UNKNOWN',
  code: 'UNKNOWN',
  syscall: 'spawn'
}
